<ui version="4.0" >
 <class>motortestClass</class>
 <widget class="QDialog" name="motortestClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>motortest</string>
  </property>
  <widget class="QLineEdit" name="lineEdit" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>330</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>产量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>网头选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>370</x>
     <y>70</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2" >
   <property name="geometry" >
    <rect>
     <x>430</x>
     <y>70</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3" >
   <property name="geometry" >
    <rect>
     <x>490</x>
     <y>70</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>120</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>正转</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>120</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>反转</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>120</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>170</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>JOG+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>170</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>JOG-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>220</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>跟随</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>690</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text" >
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>220</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>停止跟随</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>280</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>开机允许</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>280</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>主机加速</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>280</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>主机减速</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>280</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>报错</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16" >
   <property name="geometry" >
    <rect>
     <x>410</x>
     <y>280</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>报警</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17" >
   <property name="geometry" >
    <rect>
     <x>500</x>
     <y>280</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>橡毯启动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_18" >
   <property name="geometry" >
    <rect>
     <x>590</x>
     <y>280</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>橡毯停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_19" >
   <property name="geometry" >
    <rect>
     <x>680</x>
     <y>280</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>工作量慢</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>330</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_20" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>开机请求</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_21" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>报警</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_22" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_23" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>报错</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>70</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>speed</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
